app/models/base.rb

```diff
@@ -45,6 +45,8 @@ def get_query_url(options={})
     options[:number] ||= 1
     options[:size] ||= 1000
     updated = "updated:[#{options[:from_date]}T00:00:00Z TO #{options[:until_date]}T23:59:59Z]"
+    options[:exclude_registration_agencies] ||= true
+    options[:resource_type_id] ||= ""
```

Comment:
can you explain this line.

Reply:
without excluding other registrations agencies,  the method get_data (https://github.com/datacite/levriero/blob/1a7d39c4d04b5e4a21ec47b97f5cf970e6927440/app/models/base.rb#L128) will bring all updated dois, including those from crossref.citations which might have in their metadata related identifiers to other Crossref DOIs.  That would create xref-to-xref events.